lms/migrations/versions/c349a812ca82_add_h_group_table.py

```diff
@@ -0,0 +1,36 @@
+"""
+Add h_group table.
+
+Revision ID: c349a812ca82
+Revises: 5086e8b137b9
+Create Date: 2021-04-27 15:17:16.432922
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "c349a812ca82"
+down_revision = "5086e8b137b9"
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "h_group",
+        sa.Column("created_at", sa.DateTime(), nullable=True),
+        sa.Column("updated_at", sa.DateTime(timezone=True), nullable=True),
+        sa.Column("id", sa.Integer(), autoincrement=True, nullable=False),
+        sa.Column("_name", sa.String(), nullable=False),
+        sa.Column("authority_provided_id", sa.UnicodeText(), nullable=False),
+        sa.Column("type", sa.String(), nullable=False),
+        sa.PrimaryKeyConstraint("id", name=op.f("pk__h_group")),
+        sa.UniqueConstraint("authority_provided_id", name=op.f("uq__h_group__name")),
+    )
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_table("h_group")
+    # ### end Alembic commands ###
```

Comment:
It's super nice to have migration auto-generation working.

We should probably delete the generated comments.

This migration will need to be split out into a separate PR because it needs to be deployed and run before we can deploy the Python code that uses the new model